migrations/2020/Version20200214114359.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20200214114359 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'Unify mapping between Doctrine and ac_* tables';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        $this->addSql('DROP INDEX participantId_event_type ON ac_participants');
+        $this->addSql(<<<SQL
+            ALTER TABLE ac_participants
+                CHANGE id id CHAR(36) NOT NULL COMMENT '(DC2Type:payment_id)',
+                CHANGE participantId participantId INT UNSIGNED NOT NULL,
+                CHANGE event_id event_id INT NOT NULL,
+                CHANGE payment payment NUMERIC(8, 2) NOT NULL COMMENT '(DC2Type:money)',
+                CHANGE repayment repayment NUMERIC(8, 2) NOT NULL COMMENT '(DC2Type:money)',
+                CHANGE isAccount isAccount VARCHAR(255) NOT NULL
+        SQL);
+        $this->addSql('ALTER TABLE ac_chitsCategory_object DROP FOREIGN KEY ac_chitsCategory_object_ibfk_1');
+        $this->addSql('ALTER TABLE ac_chitsCategory_object CHANGE objectTypeId objectTypeId VARCHAR(20) NOT NULL COMMENT \'(DC2Type:string_enum)\', ADD PRIMARY KEY (categoryId, objectTypeId)');
+        $this->addSql('ALTER TABLE ac_chitsCategory_object ADD CONSTRAINT FK_824C4F259C370B71 FOREIGN KEY (categoryId) REFERENCES ac_chitsCategory (id)');
+        $this->addSql('ALTER TABLE ac_chitsCategory_object RENAME INDEX categoryid TO IDX_824C4F259C370B71');
+        $this->addSql('ALTER TABLE ac_chits_item CHANGE price price DOUBLE PRECISION UNSIGNED NOT NULL');
```

Comment:
Někdy UNSIGNED odebiras a jinde pridavas, proc?

Reply:
Bral jsem to, co jsme zvolili v mapování. Postupně bych ho všude vypustil.

Reply:
a nechces ho tedy vypustit vsude rovnou?

Reply:
Ok, doplním do tohoto PR, ale jako samostatnou migraci, ať v tok nedělám ještě větší zmatek.

Reply:
Hodil jsem to nakonec do issue, ok?

https://github.com/skaut/Skautske-hospodareni/issues/1346